### PR TITLE
update notebook wg meeting

### PR DESCRIPTION
**Pull request by thesuperzapper**
This updates the Notebook WG meeting information

### CODE REVIEW

**Comment by thesuperzapper** [comment]
@Bobgy can you please review?

**Comment by thesuperzapper** [comment]
@kubeflow/wg-notebook-leads

File: calendar/calendar.yaml [hunk 1]
```diff
@@ -377,3 +377,20 @@
       Notes & Agenda: https://docs.google.com/document/d/1GHi-NFHmDA2TnH1pDrs4O7OyIQdZRtKatGAdedxte4w
       Zoom: Provided in meeting notes
   organizer: woop
+
+- id: kf034
```
Comment (Bobgy) [inline calendar/calendar.yaml:381]: Why not create your own calendar? Everything should be free.
e.g. people are already moving away from centralized calendar: https://github.com/kubeflow/community/pull/463

Comment (thesuperzapper) [reply calendar/calendar.yaml:381]: @Bobgy, we don't really have a clear place to put it right now, and this will help us get the working group going

**Comment by yanniszark** [comment]
@Bobgy I agree that we should move towards separate calendars for each WG.
@thesuperzapper thanks a lot for pushing to get the Notebook WG going, your energy is inspiring, it will be a great release going forward.
I am actually in the process of preparing the docs/cals/meetings for wg-manifests.
I am also doing the same for wg-notebooks. Arrikto will provide the infra for calls/docs/cals.
In conclusion, we can close this PR and expect the new links asap, as I will probably finish them today.

**Comment by thesuperzapper** [comment]
@yanniszark, I strongly believe we should merge this

There is no reason to intentionally disrupt the setup of the Notebook WG while we wait for other changes in the community, there is no downside to adding this meeting to kubeflow-discuss for this week.

@jtfogarty can you please approve this and sync the calendar?

**Comment by yanniszark** [comment]
@thesuperzapper I think this is not true:
> There is no reason to intentionally disrupt the setup of the Notebook WG while we wait for other changes in the community, there is no downside to adding this meeting to kubeflow-discuss for this week.
1. There is no disruption to the setup of the Notebooks WG. We have created all the necessary items (doc, cal, meet) which you can see listed <here> (comment with our new links in the WG status check issue).
2. Adding the meeting to this calendar and then moving to a different one would surely cause confusion, compared to the scenario where we use our own calendar from the start.

Finally, about this:
> @jtfogarty can you please approve this and sync the calendar?

Let's avoid merging before we have reached an agreement :)
After all, this is a matter of the notebooks wg and should be decided by its [leads](https://github.com/kubeflow/community/blob/master/wgs.yaml#L227-L256).
I understand the eagerness to move forward, so I propose using the cal/docs/meet we created and have listed https://github.com/kubeflow/community/issues/459#issuecomment-739885908.
It will be the most future-proof way to move forward and is aligned with the direction that the KF community is going (see https://github.com/kubeflow/community/pull/463).

**Comment by thesuperzapper** [comment]
@yanniszark, the main thing I am trying to avoid is a lack of participation, currently the best way to encourage participation is add an event to the `kubeflow-discuss` calendar.

As a prime example of what I am talking about, your calendar link (https://arrik.to/kf-wg-manifests-cal) is not even publicly accessible.

Also, I don't see why you felt the need to make a seperate notes GoogleDoc (only a day after I started this process), the (notes I provided are stored on the Kubeflow org, so you can easily be made an editor of them)?

I see no reason why we shouldn't just merge this, and get the working group going, (we are going to have to change the invite to be bi-weekly after a few weeks anyway).

**Comment by thesuperzapper** [comment]
Additionally, we need to merge this so that the meeting times are up to date in the `wgs.yaml`

**Comment by Bobgy** [comment]
> Additionally, we need to merge this so that the meeting times are up to date in the `wgs.yaml`, as we already agreed in the slack channel `wg-notebooks` on the time `Thursdays @ 9am EET`

Can you remove the calendar.yaml change, so that I can approve?

**Comment by thesuperzapper** [comment]
@Bobgy please look at the new PR: https://github.com/kubeflow/community/pull/464

**Pull request closed without merging**